src/main/resources/db/changelog/changes/20240725-bv.yaml

```diff
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: 2024-07-25-1
+      author: Victor
+      changes:
+        - renameColumn:
+            tableName: users
+            oldColumnName: email
+            newColumnName: primary_email
+            columnDataType: VARCHAR(255)
+        - addColumn:
+            tableName: users
+            columns:
+              - column:
+                  name: phone_number
```

Comment:
nu ai consistenta cu atributu din entitate, acolo ai zis ca trebuie sa fie si unic, iar aici ai zis ca are 15 caractere iar in model n ai zis nimic de lungime

Reply:
ochi de soacra... am pus constrangererea de unicitate si in liquibase si am pus si length pe coloana phoneNumbers de 15, pe restul n am pus pentru ca am vazut ca ele oricum sunt 255 by default